R/driver.R

```diff
@@ -137,7 +148,8 @@ driver <- function(all_data = empty_data(), write_outputs = TRUE, quiet = FALSE,
       if(!quiet) print(paste("- make", format(round(tdiff, 2), nsmall = 2)))
 
       check_chunk_outputs(chunk, chunk_data, input_names,
-                          promised_outputs = subset(chunkoutputs, name == chunk)$output)
+                          promised_outputs = subset(chunkoutputs, name == chunk)$output,
+                          outputs_xml = subset(chunkoutputs, name == chunk)$to_xml)
```

Comment:
I don't know if it'll ever happen, but I think the same was it's useful visually to see the `FILE =` callout, it's also useful to see `XML =`.